doc/holochain_101/src/bridging.md

```diff
@@ -1 +1,81 @@
-# (E) Building Holochain Apps: Bridging
+# Building Holochain Apps: Bridging
+
+As you saw in [Building Apps](./building_apps.md) each **DNA** has a unique Hash, that spawns a brand new **DHT network** and creates isolated **source chains** for each agent. Even when you change the DNA, releasing a new version of the App, it will spawn a brand new DHT network and source chains.
```

Comment:
```suggestion
As you saw in [Building Apps](./building_apps.md) each **DNA** has a unique Hash that spawns a brand new **DHT network** and creates isolated **source chains** for each agent. Even when you change the DNA, releasing a new version of the App, it will spawn a brand new DHT network and source chains.
```